api/v1beta2/foundationdb_status.go

```diff
@@ -140,6 +140,21 @@ type FoundationDBStatusClusterInfo struct {
 
 	// BounceImpact represents the bounce_impact part of the machine-readable status.
 	BounceImpact FoundationDBBounceImpact `json:"bounce_impact,omitempty"`
+
+	// DatabaseAvailable indicates whether the database is currently available.
+	DatabaseAvailable bool `json:"database_available,omitempty"`
```

Comment:
I would prefer using a pointer to a bool here, that makes it easier to differentiate between the value being set intentionally and the default value. That applies to the other fields too. I know that this means some additional work when using those values but in some cases it might be helpful to know if the value was set or not.